projects/lendflare/index.js

```diff
@@ -0,0 +1,23 @@
+const sdk = require('@defillama/sdk');
+const convexBooster = require('./convexBooster.js');
+const supplyBooster = require('./supplyBooster.js');
+const uniswap = require('./uniswap.js');
+const { toUSDTBalances } = require('./utils.js');
+
+async function tvl(timestamp, block, chainBlocks) {
```

Comment:
this is the `block` variable that I was talking about, can you forward to tvl calls and use it